### ISSUE TITLE
Ability to change font settings for EACH heading, in the BB theme settings.

### ISSUE THREAD

**Issue by Clark-AG** [beaverbuilder/feature-requests#147]
I would like to be able to set a different font for each heading.

The reason for this is because when font size is the ONLY difference between headings, it makes it hard to break up a long blog post nicely. 

Also it allows you to create a unique typography system on your site. Most sites with great typography have their h3's in a different font than their h1's and h2's.

I have used the Bridge theme in the past and it allows you to set every single setting for each heading. Font, line height, letter spacing, etc.

I would like to see this in BB!

**Comment by iamjohnwhite** [comment]
Agreed! Design team mentioned this in our meeting today as well. Having at least the option for h1, h2, and h3 to differ would be great so you can have a nice combination of styles throughout a page.
